src/requests/Request.tsx

```diff
@@ -88,14 +91,32 @@ class Request extends PureComponent<RequestStatus> {
           <RequestLabel>{Labels.status}</RequestLabel>
           <RequestContent>
             {
-              errorCode
-                ?
+              errorCode ?
                 this.codeMapper(errorCode.toString())
                 :
                 <Loader>Loading...</Loader>
             }
           </RequestContent>
         </RequestTableCell>
+        <RequestTableCell>
+          <RequestLabel>{Labels.value}</RequestLabel>
+          <RequestContent>
+            {
+              hash ?
+                <LinkWrapper
+                  href={`https://ropsten.etherscan.io/tx/${hash}`}
```

Comment:
Put urls in const/configs not inside views.

Reply:
Is it always be a ropsten? I think this shluld be taken from ENV variable. You can use ropsten or other provider.

Reply:
we can use only ropsten.

Reply:
done